src/base/bittorrent/torrentinfo.cpp

```diff
@@ -76,13 +76,33 @@ TorrentInfo TorrentInfo::load(const QByteArray &data, QString *error) noexcept
 
 TorrentInfo TorrentInfo::loadFromFile(const QString &path, QString *error) noexcept
 {
+    if (error)
+        error->clear();
+
+    QFile file {path};
+    file.open(QIODevice::ReadOnly);
+    const QByteArray data = file.read(100 * 1024 * 1024);  // 100 MB
```

Comment:
It needs also to check QFile open/read for errors. Otherwise you can return meaningless error message (via error param).

Reply:
done.